pkg/storage/tpr/storage_interface_test.go

```diff
@@ -348,7 +363,9 @@ func TestGetWithNoNamespace(t *testing.T) {
 
 func TestGetWithNamespace(t *testing.T) {
 	keyer := getInstanceKeyer()
-	fakeCl := fake.NewRESTClient()
+	fakeCl := fake.NewRESTClient(func() runtime.Object {
```

Comment:
it wouldn't make much of an improvement, technically or otherwise, would it? we would still gonna need to create the functions like this